kolibri/core/tasks/registry.py

```diff
@@ -298,6 +298,19 @@ def enqueue(self, job=None, retry_interval=None, priority=None, **job_kwargs):
             retry_interval=retry_interval,
         )
 
+    def enqueue_lifo_job(self, job=None, retry_interval=None, priority=None, **job_kwargs):
```

Comment:
Just for consistency with the other methods on this class, could we change this to `enqueue_lifo` also?